src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Attorney"
+        },
+        "Witness": {
+            "type": "object",
+            "description": "The Witness to the signing of the LPA.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                },
+                "signature": {
+                    "$ref": "#/definitions/Signature"
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Witness"
+        },
+        "CertificateProvider": {
+            "type": "object",
+            "description": "The Certificate Provider on the LPA.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                },
+                "signature": {
+                    "$ref": "#/definitions/Signature"
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Certificate Provider"
+        },
+        "PersonToNotify": {
+            "type": "object",
+            "description": "The Person to Notify.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Person To Notify"
+        },
+        "ReplacementAttorney": {
+            "type": "object",
+            "description": "The Replacement Attorney.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Replacement Attorney"
+        },
+        "Actor": {
+            "type": "object",
+            "description": "Generic Actor that can be inherited across definitions.",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "title": "Actor unique identifier",
+                    "description": "Identifies the Actor with a unique identifier.",
+                    "type": "integer",
+                    "format": "^[0-9]+$"
+                },
+                "title": {
```

Comment:
Should this refer to a lookup of some kind, but still allow free-form text if necessary?